src/js/state/Spaces/types.js

```diff
@@ -49,31 +50,31 @@ export type SPACES_DETAIL = {
 export type SPACES_INGEST_PROGRESS = {
   type: "SPACES_INGEST_PROGRESS",
   clusterId: string,
-  name: string,
+  spaceID: string,
```

Comment:
In the other places in the app we have, like right above this line here `clusterId` with a lower-case 'd'. It's probably a nuisance to change, but I think I prefer `spaceId` the lower case 'd' since it conforms to the rule (rather than being an exception like JSON.parse) and matches other variables like 'clusterId'.  It also matches the common DOM api "document.getElementById". 

I personally would love everything lower-cased with underscores, but javascript... or rather...JavaScript 🤦🏼‍♂️

Reply:
Haha, sure thing. I guess this is me trying to make js more like go, where acronyms and abbreviations like ID are capitalized within camelcase. But I recognize that's not appropriate here. Will adjust.